### PR TITLE
Seperate Tizen.Wearable.CircualrUI to new API subject

### PR DESCRIPTION
**Pull request by jkpu**
### Change Description ###
Seperate Tizen.Wearable.CircualrUI API description to new API subject.
- previous CircularUI description is only one line and link.
- For supporting Gear watch .NET application.  CircularUI API link should be more easily found.
  so seperate CircularUI description to new API subject.

### Bugs Fixed ###

N/A

### API Changes ###

N/A

### CODE REVIEW

**Comment by mijin85cho** [comment]
Hello, @annie-samsung 
Would you please review PR a little bit faster?

This PR is related with opening of gear 4,
so it is requested to be published faster.
As there are not so much updates, I hope we can finish the review of this PR until tomorrow.

File: docs/application/dotnet/api/overview.md [hunk 1]
```diff
@@ -24,7 +25,11 @@ The Tizen.NET contains a few limitations on the Xamarin.Forms APIs; these limita
 
 A list of limitations is available at [Current Xamarin.Forms limitations](xamarin.forms-limitations.html).
 
-You can efficiently add Tizen wearable device specific user interfaces. The extended details for these APIs are available on the [Tizen.Wearable.CircularUI](https://github.com/Samsung/Tizen.CircularUI/).
+## Tizen.Wearable.CircularUI
+
+[Tizen.Wearable.CircularUI](https://samsung.github.io/Tizen.CircularUI/index.html) provides Tizen wearable-specific user interfaces. Tizen.Wearable.CircularUI is an extension API of Xamarin.Forms. But these APIs only support Tizen wearable devices which support Tizen .NET, unlike Xamarin.Forms which supports cross platforms.
+
+You can efficiently add Tizen wearable-specific interfaces to your Tizen wearable application. The extended details for these APIs are available on the [Tizen.Wearable.CircularUI](https://github.com/Samsung/Tizen.CircularUI).
```
Comment (annie-samsung) [inline docs/application/dotnet/api/overview.md]: **Please ignore this comment if it is not in the scope of this review:**

This page refers to the APIs as: Tizen.Wearable.CircularUI 
The API reference page title is: Tizen CircularUI
The Github page intro title: Tizen Wearable CircularUI (repo name seems Tizen.CircularUI)
Should we consider to refer the API title conistently?

Also on the Github page: I happened to notice some typo:
Tizen Wearable CircularUI project is to develop an open source software motivate software developer to creating Tizen Wearable Xamarin Forms app more easily and efficiently.

Suggestion: (Not sure if I have understood the intention clearly, please verify)
Tizen Wearable CircularUI project is an open source software to motivate and help application developers to create Tizen Wearable Xamarin Forms apps more easily and efficiently.

**Comment by mijin85cho** [comment]
Hello, @jkpu
I have a question : 
In the **Change Description**, it is said that Tizen.Wearable.CircualrUI API description is separated to be found easily.
And how about the guide? Are you preparing .NET CircularUI guide also?

**Comment by jkpu** [comment]
We will add CircularUI guide also.

File: docs/application/dotnet/api/overview.md [hunk 2]
```diff
@@ -24,7 +25,11 @@ The Tizen.NET contains a few limitations on the Xamarin.Forms APIs; these limita
 
 A list of limitations is available at [Current Xamarin.Forms limitations](xamarin.forms-limitations.html).
 
-You can efficiently add Tizen wearable device specific user interfaces. The extended details for these APIs are available on the [Tizen.Wearable.CircularUI](https://github.com/Samsung/Tizen.CircularUI/).
+## Tizen.Wearable.CircularUI
+
+[Tizen.Wearable.CircularUI](https://samsung.github.io/Tizen.CircularUI/index.html) provides Tizen wearable-specific user interfaces. It is a set of extension APIs of Xamarin.Forms. Tizen.Wearable.CircularUI APIs are supported only on Tizen wearable devices with Tizen .NET support, unlike the Xamarin.Forms which support cross platforms.
+
+Using these APIs, You can easily and efficiently add Tizen wearable-specific user interfaces to your Tizen wearable application. For more information about these APIs, see [Tizen.Wearable.CircularUI GitHub](https://github.com/Samsung/Tizen.CircularUI).
```
Comment (annie-samsung) [inline docs/application/dotnet/api/overview.md]: Change You to you

Comment (jkpu) [reply docs/application/dotnet/api/overview.md]: Done

**Comment by mijin85cho** [comment]
Thanks for the check & review.

**Pull request merged**